src/resolution.js

```diff
@@ -25,6 +32,8 @@ var getScreenTypeObject = function (type) {
 * @module enyo/resolution
 */
 var ri = module.exports = {
+	config: {},
```

Comment:
Do we need this if we're assigning `config` to the default config object, further below?